src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,61 @@
+= Capturing user input
+:page-slug: capture-user-input
+:page-description: How to capture user input using `Koji.ui.capture` controls.
+:includespath: ../_includes
+
+Koji provides a set of predefined user-input controls that you can use in your app's front end to capture information and content from your users.
```

Comment:
We have standardized on “frontend”, one word. Please update throughout. This usage is already common in developer docs (e.g. https://kubernetes.io/docs/tasks/access-application-cluster/connecting-frontend-backend/,  https://about.gitlab.com/job-families/engineering/development/frontend/). I believe it will go the way of email and website, where it becomes the de facto standard over time.